beacon-chain/core/epoch/epoch_processing_test.go

```diff
@@ -499,9 +499,8 @@ func TestProcessHistoricalDataUpdate(t *testing.T) {
 				require.NoError(t, err)
 				require.DeepEqual(t, r[:], roots[0])
 
-				summaries, err := st.HistoricalSummaries()
-				require.NoError(t, err)
-				require.Equal(t, 0, len(summaries))
+				_, err = st.HistoricalSummaries()
+				require.ErrorContains(t, "HistoricalSummaries is not supported for phase0", err)
```

Comment:
can we use `require.ErrorIs` instead of string matching the error text?

Reply:
Error is not exported. I can export the error though. Was trying to minimize the changes since the diff is getting big